4.queueRange.js

```diff
@@ -35,7 +35,12 @@
 
 // Write your function here
 const range = (queue) => {
+    let newValue = 0
+    while(!queue.isEmpty()){
+        newValue += queue.dequeue();
 
+    }
+    return newValue
 }
 
```

Comment:
this problem needs you to find a max value and a min value. At the end you'd return the difference max - min.